src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -723,7 +732,8 @@ def rsa_padding_supported(self, padding: AsymmetricPadding) -> bool:
         if isinstance(padding, PKCS1v15):
             return True
         elif isinstance(padding, PSS) and isinstance(padding._mgf, MGF1):
-            # SHA1 is permissible in MGF1 in FIPS
+            # SHA1 is permissible in MGF1 in FIPS and when SHA1 is blocked
```

Comment:
is this supposed to say "even when" rather than "and when"?

Reply:
Yes ...